docs/source/admin/traffic_ops.rst

```diff
@@ -0,0 +1,706 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. role:: bash(code)
+	:language: bash
+
+***********
+Traffic Ops
+***********
+Traffic Ops is quite possible the single most complex and most important Traffic Control component. It has many different configuration options that affect a wide range of other components and their interactions.
+
+.. _to-install:
+
+Installing
+==========
+
+System Requirements
+-------------------
+The user must have the following for a successful minimal install:
+
+- CentOS 7+
+- Two machines - physical or virtual -, each with at least two (v)CPUs, 4GB of RAM, and 20 GB of disk space
+- Access to CentOS Base and EPEL :manpage:`yum(8)` repositories
+- Access to `The Comprehensive Perl Archive Network (CPAN) <http://www.cpan.org/>`_
+
+Guide
+-----
+#. Install PostgreSQL Database. For a production install it is best to install PostgreSQL on its own server/virtual machine.
+
+	.. seealso:: For more information on installing PostgreSQL, see `their documentation <https://www.postgresql.org/docs/>`_.
+
+	.. code-block:: shell
+		:caption: Example PostgreSQL Install Procedure
+
+		yum update -y
+		yum install -y https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
+		yum install -y postgresql96-server
+		su - postgres -c '/usr/pgsql-9.6/bin/initdb -A md5 -W' #-W forces the user to provide a superuser (postgres) password
+
+
+#. Edit :file:`/var/lib/pgsql/9.6/data/pg_hba.conf` to allow the Traffic Ops instance to access the PostgreSQL server. For example, if the IP address of the machine to be used as the Traffic Ops host is ``192.0.2.1`` add the line ``host  all   all     192.0.2.1/32 md5`` to the appropriate section of this file.
+
+#. Edit the :file:`/var/lib/pgsql/9.6/data/postgresql.conf` file to add the appropriate listen_addresses or ``listen_addresses = '*'``, set ``timezone = 'UTC'``, and start the database
+
+	.. code-block:: shell
+		:caption: Starting PostgreSQL with :manpage:`systemd(1)`
+
+		systemctl enable postgresql-9.6
+		systemctl start postgresql-9.6
+		systemctl status postgresql-9.6 # Prints the status of the PostgreSQL service, to prove it's running
+
+
+#. Build a :file:`traffic_ops-{version string}.rpm` file using the instructions under the :ref:`dev-building` page - or simply download a pre-built `release <https://github.com/apache/trafficcontrol/releases>`_.
```

Comment:
I dont see any pre-built rpms attached to the releases which is actually very surprising.

Reply:
That is odd. I know we had downloads for those somewhere, but I can't find them.